tdvt/tdvt/tdvt.py

```diff
@@ -132,8 +142,8 @@ def copy_test_result_file(self):
     def copy_files_and_cleanup(self):
         left_temp_dir = False
         try:
-            self.copy_files_to_zip(TestOutputFiles.output_actuals, self.temp_dir, 'actual.*')
-            self.copy_files_to_zip(TestOutputFiles.output_tabquery_log, self.temp_dir, '*/all_logs.zip')
```

Comment:
So does this mean you can delete this code now? https://github.com/tableau/connector-plugin-sdk/blob/dev/tdvt/tdvt/tdvt_core.py#L175

Reply:
I'm glad to! It's been removed in the commit below.